calibrations/sepd/eventplane/scripts/runProd.py

```
@@ -0,0 +1,872 @@
+
+    Returns:
+        bool: True if the command returned exit code 0, False otherwise.
+    """
+    if do_logging:
+        logger.info(f"{description}: '{command}'")
+
+    try:
+        # Using shell=False is generally safer, but if you need pipes (|),
+        # keep shell=True or use python logic.
+        # Here we use shell=True because your split command uses pipes.
+        result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)